### ISSUE TITLE
Provision nginx and letsencrypt

### ISSUE THREAD

**Issue by davewsmith** [davewsmith/homelab#3]
Assuming tailscale configured

- get a letsencrypt cert and put it somewhere (/etc/ssl/private ?)
- set up a cron or systemd job to refresh the cert. https://blog.viktorpetersson.com/2022/12/23/securing-services-with-tailscale.html has a recipe.
- install nginx if needed
- emplace an nginx enabled-site that uses the cert and shunts http: to https:
- drop in placeholder HTML (if needed)
- restart nginx if needed

A config for uwsgi is out of scope.

**Comment by davewsmith** [comment]
Blocked by #10

**Comment by davewsmith** [comment]
Adapted https://sleeplessbeastie.eu/2023/02/20/how-to-use-tailscale-to-generate-certificate/ to auto-renew the cert on pinot. It checks weekly, and puts certs in /etc/nginx/

**Comment by davewsmith** [comment]
This has been working satisfactorily for a while now

**Comment by davewsmith** [comment]
Closing this as 'good enough for now'
